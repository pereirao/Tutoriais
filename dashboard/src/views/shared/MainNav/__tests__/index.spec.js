import React from 'react';

// NOTE: not the connected one
import { MainNav } from '../index';

it('renders correctly', () => {
  const stubbed_location = {pathname: '/'};
  expect(
    shallow(<MainNav location={stubbed_location} ping={() => {}} />)
  ).toMatchSnapshot();
});
