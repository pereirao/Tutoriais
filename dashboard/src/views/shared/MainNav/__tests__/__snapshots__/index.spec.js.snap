// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="navWrapper"
>
  <AudioPermissionModal
    alertNotification={[Function]}
    closeModal={[Function]}
    hidden={true}
  />
  <TopBar
    current_path="/"
  />
  <div
    className="navMenu"
  >
    <MainNavLink
      current_path="/"
      route_name="orders"
    />
    <MainNavLink
      current_path="/"
      route_name="reports"
    />
    <MainNavLink
      current_path="/"
      route_name="inventory"
    />
    <MainNavLink
      current_path="/"
      route_name="settings"
    />
    <Connect(StoreRating) />
  </div>
</div>
`;
